### ISSUE TITLE
Form List Page Count Error

### ISSUE THREAD

**Issue by andypullen** [verbb/formie#2127]
### Describe the bug

When viewing the form list page, displaying the Page Count column and then sorting by Page Count results in a notification error, and then the list will not render and the display settings get stuck.

### Steps to reproduce

1. Go to the Form list page
2. Click View and select the Page Count in the Table Columns
3. Sort by Page Count
You'll then get an error notification (shown below) and will be unable to change the View settings and the form list will not complete rendering.

### Form settings

- Multi-page form: N/A
- Submission Method: N/A
- Client-side Validation: N/A
- Custom Form Templates: No

### Craft CMS version

5.4.10 Pro

### Plugin version

3.0.9

### Multi-site?

Yes

### Additional context

The notification displays this text:
```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'pageCount' in 'order clause' The SQL being executed was: SELECT `elements`.`id`, `elements`.`canonicalId`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateLastMerged`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`id` AS `siteSettingsId`, `elements_sites`.`siteId`, `elements_sites`.`title`, `elements_sites`.`slug`, `elements_sites`.`uri`, `elements_sites`.`content`, `elements_sites`.`enabled` AS `enabledForSite`, `formie_forms`.`id`, `formie_forms`.`handle`, `formie_forms`.`settings`, `formie_forms`.`layoutId`, `formie_forms`.`templateId`, `formie_forms`.`submitActionEntryId`, `formie_forms`.`submitActionEntrySiteId`, `formie_forms`.`defaultStatusId`, `formie_forms`.`dataRetention`, `formie_forms`.`dataRetentionValue`, `formie_forms`.`userDeletedAction`, `formie_forms`.`fileUploadsAction`, `formie_forms`.`uid` FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `siteSettingsId` FROM `craft_elements` `elements` INNER JOIN `craft_formie_forms` `formie_forms` ON `formie_forms`.`id` = `elements`.`id` INNER JOIN `craft_elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id` WHERE (`elements_sites`.`siteId`=2) AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`elements`.`draftId` IS NULL) AND (`elements`.`revisionId` IS NULL) ORDER BY `elements_sites`.`title`, `pageCount` LIMIT 100) `subquery` INNER JOIN `craft_elements` `elements` ON `elements`.`id` = `subquery`.`elementsId` INNER JOIN `craft_elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`siteSettingsId` INNER JOIN `craft_formie_forms` `formie_forms` ON `formie_forms`.`id` = `subquery`.`elementsId` ORDER BY `elements_sites`.`title`, `pageCount`
```
![Screen Shot 2024-11-07 at 16 53 47@2x](https://github.com/user-attachments/assets/62224c0c-a43e-4f4f-a707-a62626fcbe49)

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.9"`.

**Comment by engram-design** [comment]
Fixed in [3.0.10](https://github.com/verbb/formie/releases/tag/3.0.10)
